samples/net/wifi/sample.yaml

```diff
@@ -0,0 +1,8 @@
+sample:
```

Comment:
I did not notice that you had added sample.yaml file for wifi, so I have similar addition for that in #10405

Reply:
OK, then I'll remove this one.